src/eos-crash-metrics.c

```diff
@@ -41,6 +70,18 @@ report_crash (const char *binary,
   g_variant_dict_insert_value (&dict, "ostree_commit", g_variant_new_string (ostree_commit));
   g_variant_dict_insert_value (&dict, "ostree_url", g_variant_new_string (ostree_url));
 
+  if (info != NULL)
```

Comment:
If you follow my suggestion below, `info` will never be `NULL` when this function is called.

Reply:
It can be null, if we have a non flatpak executable. I left this check in but addressed the failure handling when the flatpak information can not be retrieved.